pnr/openroad/meta.yaml

```diff
@@ -0,0 +1,59 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('openroad-', '')|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: openroad
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/The-OpenROAD-Project/OpenROAD.git
+    git_rev: master
+    patches:
+      - odb-drt-disable-tests.patch
+      - sta-add-install-options.patch
+      - gui-find-qt-only-if-enabled.patch
+
+build:
+  # number: 202202031935
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20220203_1935
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+
+requirements:
+  build:
+    - cmake
+    - swig
+    - bison
+    - doxygen
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
```

Comment:
we should use the same version used here: https://github.com/The-OpenROAD-Project/OpenROAD/blob/af1f38c9f23493ae1f604566066e2b9d24e336b2/etc/DependencyInstaller.sh#L5